rules-tests/CodeQualityStrict/Rector/If_/MoveOutMethodCallInsideIfConditionRector/Fixture/if_condition_method_call_with_namespaced_function_call.php.inc

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rector\Tests\CodeQualityStrict\Rector\If_\MoveOutMethodCallInsideIfConditionRector\Fixture;
+
+use function Symfony\Component\String\s;
+
+class IfConditionMethodCallWithNamespacedFunctionCall
+{
+    public function check($check)
+    {
+        if (s('test')->bytesAt(0)) {
```

Comment:
I think it need another test case for backslashed function, eg:

```php
if (s\bar\baz('test')->bytesAt(0)) {
```

Reply:
I tested locally and it seems working ok for namespaced function call directly 👍